.circleci/config.yml

```diff
@@ -22,7 +22,6 @@ jobs: # Defines the overall Jobs for this particular pipeline to be used in work
       - image: circleci/openjdk:11-jdk
     steps:
       - setup_remote_docker:
```

Comment:
You gotta remove the `:` as well, that's the issue the build is having now i think.

Reply:
fixed